src/Microsoft.AspNetCore.Testing/xunit/FrameworkSkipConditionAttribute.cs

```diff
@@ -33,24 +33,23 @@ private static bool CanRunOnThisFramework(RuntimeFrameworks excludedFrameworks)
                 return true;
             }
 
+#if NET451
             if (excludedFrameworks.HasFlag(RuntimeFrameworks.Mono) &&
                 TestPlatformHelper.IsMono)
             {
                 return false;
             }
 
-            if (excludedFrameworks.HasFlag(RuntimeFrameworks.CLR) &&
-                PlatformServices.Default.Runtime.RuntimeType.Equals("CLR", StringComparison.OrdinalIgnoreCase))
+            if (excludedFrameworks.HasFlag(RuntimeFrameworks.CLR))
             {
                 return false;
             }
-
-            if (excludedFrameworks.HasFlag(RuntimeFrameworks.CoreCLR) &&
-                PlatformServices.Default.Runtime.RuntimeType.Equals("CoreCLR", StringComparison.OrdinalIgnoreCase))
+#else
```

Comment:
Just because we compiled for .NET Standard doesn't mean this code will run on CoreCLR only. .NET Standard 1.3 can run on CoreCLR, .NET Framework, and Mono.